doc/selfhosting.md

```diff
@@ -123,23 +123,31 @@ API. Generate one randomly with `pwgen -s 30 1`.
 ## Build AMI
 
 You'll want to run `make env` to load in the new variables from
-`.env`. Now run `make packer`. This will take up to 10 minutes to
-build a timestamped AMI with a name like `riju-20210711223158`.
+`.env`. Now run `make packer-ci`. This will take up to 5 minutes to
+build a timestamped AMI with a name like `riju-ci-20210711223158`.
+Then run `make packer-web`. This will take up to 10 minutes to build
+another AMI with a name like `riju-web-20210711223158.`
```

Comment:
The CI isn't actually quite working yet, see https://github.com/raxod502/riju/issues/80 for tracking that. I've fixed the Terraform so that it doesn't require `CI_AMI_NAME`.

Reply:
I think a few things in tf/iam.tf are requiring ci? `data.aws_ami.ci.arn`

Reply:
My mistake, should be fixed in https://github.com/raxod502/riju/commit/a1815f8dd912de831b29a396c3bbf86f373a3e0a.